### ISSUE TITLE
Can't create a blank slide - is information stored somewhere?

### ISSUE THREAD

**Issue by cansavvy** [hairizuanbinnoorazman/rgoogleslides#32]
### Summary 
I was doing quite a bit of fiddling with creating image requests and now can't create a new blank slide. 

I unfortunately don't know how to give an exactly reproducible example for this problem because it is probably related to my account. 

Whenever I run code to create a new slide (see code I run below), a new slide IS created, but my new slides always have the images I was using for testing before even though I no longer link to them or provide it to the code. 

This is what I'm running: 
```
slides_id <- "slide_id here"
requests <- add_create_slide_page_request(predefined_layout = "BLANK") 
commit_to_slides(slides_id, requests)
``` 

And this is what shows up in my slides even though I'm just looking for (and expecting) a blank slide.

<img width="907" alt="Screen Shot 2021-04-23 at 9 35 07 AM" src="https://user-images.githubusercontent.com/23458084/115879159-5ad45500-a417-11eb-898b-caf3856db19d.png">

The images in the background of this slide are not deletable or movable if that helps narrow in on the problem. 

Things I've tried to refresh everything and get rid of those background slide images: 
  
- I've refreshed RStudio sessions many a time
- I've completely restarted my docker container I was working from
- I created a new google slide set to work from 
- I uninstalled rgoogleslides and reinstalled and tried outside of a docker container.

But no matter what, new slides have background images like that. 

Is there somewhere in the API itself or in the package that a cache is stored? How do I truly clear everything out so I can get a blank slide again?  

Any advice you can give would be great. I am stumped.

**Comment by hairizuanbinnoorazman** [comment]
I haven't dug too deep into the codebase yet but my hypothesis is that maybe the `requests` variable is being reused over again? 

If an add image to slide requests has been put in, that request will be reprocessed once more -> so there should be another new slide with the background image and another new blank slide (I haven't confirm this - will need to check the codebase)

I will double check the code here

**Comment by cansavvy** [comment]
> If an add image to slide requests has been put in, that request will be reprocessed once more -> so there should be another new slide with the background image and another new blank slide
> 
This is not the behavior I've been experiencing currently with the code above. I get one new slide but with the background images.

**Comment by hairizuanbinnoorazman** [comment]
I'm unable to replicate the issue on my side

Is it possible to get the `requests` parameter when you run the code?
It should be showing this:

```R
> requests$requests
[[1]]
[[1]]$createSlide
[[1]]$createSlide$slideLayoutReference
[[1]]$createSlide$slideLayoutReference$predefinedLayout
[1] "BLANK"

```

This is before you would run the code to commit it to the slides api

Also, before running the script to run the code, can you also clear out all variables that has been stored in your environment.
![image](https://user-images.githubusercontent.com/4027444/115970596-00053f80-a576-11eb-8066-045acbbd517a.png)

Best to have  this before starting any checks
![image](https://user-images.githubusercontent.com/4027444/115970611-13b0a600-a576-11eb-8db6-f07712008d43.png)

**Comment by cansavvy** [comment]
> Is it possible to get the `requests` parameter when you run the code?
> It should be showing this:

I do get this output before committing. But then when I run commit, the new slide created still has the image on it. 

> ```r
> > requests$requests
> [[1]]
> [[1]]$createSlide
> [[1]]$createSlide$slideLayoutReference
> [[1]]$createSlide$slideLayoutReference$predefinedLayout
> [1] "BLANK"
> ```
> 
> This is before you would run the code to commit it to the slides api
> 
> Also, before running the script to run the code, can you also clear out all variables that has been stored in your environment.
> ![image](https://user-images.githubusercontent.com/4027444/115970596-00053f80-a576-11eb-8066-045acbbd517a.png)

I've cleared out all variables many times and did it again this time too. 

Is there somewhere in the Google API itself I can check if something is being stored? 

Thanks for trying to help! I appreciate it!

**Comment by cansavvy** [comment]
I FOUND IT! I didn't make these changes using the GUI, so I don't know how it happened, but my blank slide for my theme had the plots added. Somehow I ran something that added the images to the blank template. 

I still don't know what code I ran to add to the theme, but I deleted it from the google slides website and now its fixed. 

<img width="710" alt="Screen Shot 2021-04-26 at 9 35 32 AM" src="https://user-images.githubusercontent.com/23458084/116091549-f0b7ec00-a672-11eb-9fe6-b4b6734bd359.png">

Thanks for your help!
